server/routes/referrals/referralsController.ts

```diff
@@ -130,7 +130,11 @@ export default class ReferralsController {
   async viewReferralForm(req: Request, res: Response): Promise<void> {
     const referral = await this.interventionsService.getDraftReferral(res.locals.user.token, req.params.id)
 
-    const presenter = new ReferralFormPresenter(referral)
+    const serviceCategoryName = referral.serviceCategoryId
+      ? (await this.interventionsService.getServiceCategory(res.locals.user.token, referral.serviceCategoryId)).name
+      : 'Accommodation'
```

Comment:
Is there a real-life situation where `serviceCategoryId` wouldn't have been set on a draft referral at this stage (I think not)? If so, it might be clearer to throw an error here instead of an arbitrary name.

Reply:
Yep, good point, I think I did this out of laziness...